hack/externalnode/install-vm.sh

```diff
@@ -47,26 +49,39 @@ AGENT_LOG_DIR="/var/log/antrea"
 AGENT_CONF_PATH="/etc/antrea"
 OVS_BRIDGE="br-int"
 OVS_VSWITCHD="ovs-vswitchd.service"
+OVS_BIN="start_ovs"
+
+# Antrea agent files
+ANTREA_AGENT_BIN="antrea-agent"
+ANTREA_AGENT_KUBECONFIG="antrea-agent.kubeconfig"
+ANTREA_AGENT_ANTREA_KUBECONFIG="antrea-agent.antrea.kubeconfig"
+
+# Docker variables
+ANTREA_VERSION="latest"
+ANTREA_DOCKER_REGISTRY="antrea/antrea-ubuntu"
+ANTREA_DOCKER_IMAGE="${ANTREA_DOCKER_REGISTRY}:${ANTREA_VERSION}"
+DOCKER_COMPOSE_VERSION="1.29.2"
+DOCKER_COMPOSE="https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose"
+DOCKER_COMPOSE_DIR="/usr/local/bin/docker-compose"
 
-# Optional arguments
-VALIDATE_OVS_CONFIG=false
 NODE_NAME="$(hostname)"
+OS_NAME=""
 
-# List of supported OS versions, verified by antrea.
-declare -a SUPPORTED_OS=("Ubuntu 18.04" "Ubuntu 20.04" "Red Hat Enterprise Linux 8.4")
+declare -a SUPPORTED_OS=("ubuntu 18.04" "ubuntu 20.04" "rhel 8.4")
```

Comment:
SUPPORTED_OSES?

Reply:
Supported_Os only because Supported_Oses is an incorrect word.